src/platform/windows/display_vram.cpp

```diff
@@ -993,8 +1008,29 @@ namespace platf::dxgi {
     }
 
     if (frame_info.LastMouseUpdateTime.QuadPart) {
-      cursor_alpha.set_pos(frame_info.PointerPosition.Position.x, frame_info.PointerPosition.Position.y, frame_info.PointerPosition.Visible);
-      cursor_xor.set_pos(frame_info.PointerPosition.Position.x, frame_info.PointerPosition.Position.y, frame_info.PointerPosition.Visible);
+      auto set_cursor_pos = [this](gpu_cursor_t &cursor, DXGI_OUTDUPL_POINTER_POSITION pointer_position) {
+        switch (display_rotation) {
+          case DXGI_MODE_ROTATION_UNSPECIFIED:
+          case DXGI_MODE_ROTATION_IDENTITY:
+            cursor.set_pos(pointer_position.Position.x, pointer_position.Position.y, pointer_position.Visible);
```

Comment:
This part may need deeper changes in `set_pos()` itself, if cursor image is not square (can this even happen?).

Reply:
Pushed a fix, even though all the cursors I've received from the api so far were square sized.